client/menu/menu.js

```diff
@@ -13,7 +13,17 @@
 // limitations under the License.
 
 /**
- * @fileoverview Module for storing question and user data
+ * @fileoverview Adds an importable configuration file for the TIE menu page.
  */
 
-window.tieData = angular.module('tieData', []);
+window.tieMenu = angular.module('tieMenu', ['tieData']);
+
+tieMenu.controller('MenuPageController',
```

Comment:
This seems reasonable -- but, in this or a future PR, could you please pull this and the menu page HTML out into a menu-view directive in order to mimic what is done in the question page?